site/content/docs/5.3/helpers/icon-link.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: docs
+title: Icon link
+description: Quickly create stylized hyperlinks with Bootstrap Icons or other icons.
+group: helpers
+aliases: "/docs/5.3/icon-link/"
+toc: true
+added: 5.3
+---
+
+The icon link helper component modifies our default link styles to enhance their appearance and quickly align any pairing of icon and text. Alignment is set via inline flexbox styling and a default `gap` value. We stylize the underline with a custom offset and color. Icons are automatically sized to `1em` to best match their associated text's `font-size`.
+
+Icon links assume [Bootstrap Icons](https://icons.getbootstrap.com) are being used, but you can use any icon or image you like.
+
+{{< callout >}}
+Icons used here are likely to be purely decorative, which means they should be hidden from assistive technologies using `aria-hidden="true"`, as we've done in our examples. For icons that are more than decorative, provide an appropriate text alternative via `alt` or `aria-label` attribute.
```

Comment:
the suggestion to use `alt` only works when it's an `<img>` element. for `<svg>`s, you need *both* a `role="img"` and an `aria-label`. Maybe worth adding an example right here, a la `<svg ... role="img" aria-label="..."><use xlink:href="..."></use></svg>`